### PR TITLE
Add Vagrantfile + Ansible playbook

### PR DESCRIPTION
**Pull request by kmille**
Hey,

I wrote an Ansible Playbook

### CODE REVIEW

**Comment by schinken** [comment]
Really cool! I sent these to some friends wo know more about system engineering than I do. I also don't know if a separate repo is more suitable for this. 

Thanks for the PR!

File: vagrant-ansible/strichliste-backend.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+- hosts: all
+  gather_facts: False
+  become: True
+  become_user: root
+  tasks:
+      - name: install dependencies
+        action: apt pkg={{ item }} state=present
```
Comment (encbladexp) [inline vagrant-ansible/strichliste-backend.yml]: Why `action:` and not simply `apt:` here?

File: vagrant-ansible/strichliste-backend.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        with_items:
```
Comment (encbladexp) [inline vagrant-ansible/strichliste-backend.yml]: Ansible >= 2.7 has a better readable syntax. You can pass a list for `pkg=`, no need to use `with_items` like before.

**Comment by schinken** [comment]
Can you move these files maybe in `contrib/ansible` or `contrib/vagrant` (depending on what's more suitable). Also it would be cool if the install instructions are in a README.md inside the contrib/$foo directory instead of the normal install.md file

But again: Thank you very much for the PR! :)

**Comment by kmille** [comment]
Okay, I changed the things you said. But I'm still unhappy using Debian Buster (testing):

Stretch uses php version 7.0+49 
Buster uses php version 7.3+69 

The 7.0 php is too old for Symfony. We can use Buster or use a different php repo (https://deb.sury.org/)

**Comment by schinken** [comment]
LGTM! :)

**Pull request merged**